dts/bindings/clock/st,stm32-rcc.yaml

```diff
@@ -58,6 +58,15 @@ properties:
       required: false
       enum:
         - 1
+        - 2
+        - 4
+        - 8
+        - 16
+        - 32
+        - 64
+        - 128
+        - 256
+        - 512
       description: |
```

Comment:
description should be updated.
Also, it is mentionned in binding description that property clock-frequency reflects SYSCLK frequency ,
which is not the case anymore (which could be tricky for users).

Reply:
now "_Changing this prescaler is allowed_"